ArduCopter/mode_guided.cpp

```diff
@@ -721,6 +735,22 @@ void ModeGuided::accel_control_run()
         return;
     }
 
+    // if set and no pilot input for 2 sec weathervane copter into wind
+    if (allows_weathervaning()) {
+        auto_yaw.update_weathervane(wp_nav->get_roll(), wp_nav->get_pitch(), target_yaw_rate, get_alt_above_ground_cm());
+    }
+
+    // landed with positive desired climb rate, initiate takeoff
```

Comment:
fairly sure this is not meant to be here, just a rebase thing I guess.

Reply:
yip, good catch, removed.